indico/modules/events/papers/fields.py

```diff
@@ -0,0 +1,47 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.util.i18n import _
+from indico.web.forms.fields import MultipleItemsField
+
+
+class PaperReviewQuestionsField(MultipleItemsField):
```

Comment:
This is almost identical to `AbstractReviewQuestionsField` - let's try to use a common class that handles both or a base class and then a subclass that adds the `no_score` field since the only differences seem to be the name of the question model and the (non-)existence of `no_score`.

One possible way of unifying it in a single class could be a constructor like this:

    ... = ReviewQuestionsField(_('whatever'), question_model=lambda: PaperReviewQuestion(type=PaperReviewType.xxx))

and for abstracts

    ... = ReviewQuestionsField(_('whatever'), question_model=AbstractReviewQuestion,
                               extra_fields=[{'id': 'no_score', 'caption': _("Exclude from score"),
                                              'type': 'checkbox'}])